chemprop/train/metrics.py

```diff
@@ -213,6 +229,66 @@ def accuracy(targets: List[int], preds: Union[List[float], List[List[float]]], t
     return accuracy_score(targets, hard_preds)
 
 
+def recall_metric(targets: List[int], preds: Union[List[float], List[List[float]]], threshold: float = 0.5) -> float:
+    """
+    Computes the recall of a binary prediction task using a given threshold for generating hard predictions.
+
+    Alternatively, computes recall for a multiclass prediction task by picking the largest probability.
+
+    :param targets: A list of binary targets.
+    :param preds: A list of prediction probabilities.
+    :param threshold: The threshold above which a prediction is considered positive.
+    :return: The computed recall.
+    """
+    if type(preds[0]) == list:  # multiclass
```

Comment:
The Pythonic way to check type is

```suggestion
    if isinstance(preds[0], list):  # multiclass
```